### PR TITLE
Sepolia

### PR DESCRIPTION
**Pull request by conor2e**
Updated the docs to include references to the newly supported Ethereum Sepolia testnet network. Added the network and ChainID to the _Operations>Ethereum>Tools_ section.

### CODE REVIEW

File: docs/operations/ethereum/tools.md [hunk 1]
```diff
@@ -191,8 +191,9 @@ You can set your [MetaMask](https://metamask.io/) to interact through your Ether
     * Ropsten: `3`
```
Comment (akegaviar) [inline docs/operations/ethereum/tools.md:191]: Change the order please to:

* Goerli
* Sepolia
* Ropsten
* Rinkeby

File: docs/operations/ethereum/tools.md [hunk 2]
```diff
@@ -191,8 +191,9 @@ You can set your [MetaMask](https://metamask.io/) to interact through your Ether
...
     * Rinkeby: `4`
     * Goerli: `5`
+    * Sepolia: `11155111`
 
-1. Click **Save**.
+2. Click **Save**.
```
Comment (akegaviar) [inline docs/operations/ethereum/tools.md:196]: What does it render to when building locally? We are using `1.` for [numbered lists](https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html) whenever possible.

This way it's easy to slide a new step into the list without thinking much about the resulting numbers.

File: docs/operations/ethereum/tools.md [hunk 3]
```diff
@@ -813,6 +814,7 @@ where
   * Ropsten: `3`
```
Comment (akegaviar) [inline docs/operations/ethereum/tools.md:814]: Order please

* Goerli
* Sepolia
* Ropsten
* Rinkeby

**Review by akegaviar — CHANGES_REQUESTED** [review]
Ordered list and the testnet order please

File: docs/operations/ethereum/tools.md (hunk 2 above)
Comment (conor2e) [reply docs/operations/ethereum/tools.md:196]: That's strange, I didn't modify the number so it must have changed automatically when I added the content above it. Thanks for highlighting it. It resolved fine with 2, but I have changed it back to 1.

File: docs/operations/ethereum/tools.md (hunk 3 above)
Comment (conor2e) [reply docs/operations/ethereum/tools.md:814]: The list was in order of ChainID value, lowest to highest, so I just added Sepolia at the end. Including Mainnet, I have changed the order to the following as requested:

- Mainnet 1
- Goerli 5
- Sepolia 11155111
- Ropsten 3
- Rinkeby 4

File: docs/platform/join-a-public-network.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ meta:
 
 1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
 1. Under **Blockchain protocol**, select **Ethereum**.
-1. Under **Blockchain network**, select **Mainnet**, **Ropsten testnet**, **Rinkeby testnet**, or **Goerli testnet**. Click **Next**.
+1. Under **Blockchain network**, select **Mainnet**, **Ropsten testnet**, **Rinkeby testnet**, **Goerli testnet**, or **Sepolia testnet**. Click **Next**.
```
Comment (akegaviar) [inline docs/platform/join-a-public-network.md:15]: same for order here too please: goeri, sepolia, ropsten, rinkeby

File: docs/platform/supported-protocols.md [hunk 5]
```diff
@@ -15,6 +15,7 @@ Public network:
 	* Ropsten testnet
```
Comment (akegaviar) [inline docs/platform/supported-protocols.md:15]: same here for order

**Review by akegaviar — CHANGES_REQUESTED** [review]
Protocol order

File: docs/platform/supported-protocols.md (hunk 5 above)
Comment (conor2e) [reply docs/platform/supported-protocols.md]: Updated both files.

**Review by akegaviar — APPROVED** [review]

**Pull request merged**